docs/SlavaUkraini/examples/overlays/index.md

```diff
@@ -110,12 +110,6 @@ L.tileLayer('https://api.mapbox.com/styles/v1/{id}/tiles/{z}/{x}/{y}?access_toke
 }).addTo(map);
 ```
 
-Then, we'll define the geographical bounds that the video will cover. This is an instance of [`L.LatLngBounds`](/reference.html#latlngbounds), which is a rectangular shape:
-
-```
-var latLngBounds = L.latLngBounds([[32, -130], [13, -100]]);
-```
-
 {% include frame.html url="example-bounds.html" %}
```

Comment:
I believe the include should be removed as well.

Reply:
But this is the only place "example-bounds.html" is used. Should I remove the whole "example-bounds.html" file then?

Reply:
@KonstantinBiryukov Yes I think it should be fine to remove the file as well.

Reply:
removed, thanks.